src/themes/stitch.ts

```diff
@@ -39,7 +39,7 @@ const metabase: MetabaseTheme = {
     "text-tertiary": colors.lighterGrey,
     border: colors.darkGrey,
     background: colors.background,
-    "background-hover": "#4C4A48",
+    "background-hover": colors.background,
```

Comment:
This caused a bug in the "Recents" tab in the entity picker of the CreateQuestion component. The recents tab's background uses `background-hover`. It makes sense to use the same colors for both values for dark-themed apps.